.rubocop_todo.yml

```diff
@@ -35,7 +35,7 @@ Metrics/MethodLength:
 # Offense count: 1
 # Configuration parameters: CountComments, CountAsOne.
 Metrics/ModuleLength:
-  Max: 174
+  Max: 180
```

Comment:
Wonder if we shouldn't add an exclude here for `psulib_config.rb` 🤔

Reply:
180 is from `lib/traject/macros/custom.rb`

Reply:
oh right!